sklearn/neighbors/tests/test_neighbors_tree.py

```diff
@@ -190,30 +188,6 @@ def test_node_heap(nodeheap_sort, n_nodes=50):
     assert_array_almost_equal(vals[i1], vals2)
 
 
-@pytest.mark.parametrize(
-    "simultaneous_sort", [simultaneous_sort_bt, simultaneous_sort_kdt]
-)
-def test_simultaneous_sort(simultaneous_sort, n_rows=10, n_pts=201):
-    rng = check_random_state(0)
-    dist = rng.random_sample((n_rows, n_pts)).astype(DTYPE, copy=False)
-    ind = (np.arange(n_pts) + np.zeros((n_rows, 1))).astype(ITYPE, copy=False)
-
-    dist2 = dist.copy()
-    ind2 = ind.copy()
-
-    # simultaneous sort rows using function
-    simultaneous_sort(dist, ind)
-
-    # simultaneous sort rows using numpy
-    i = np.argsort(dist2, axis=1)
-    row_ind = np.arange(n_rows)[:, None]
-    dist2 = dist2[row_ind, i]
-    ind2 = ind2[row_ind, i]
-
-    assert_array_almost_equal(dist, dist2)
-    assert_array_almost_equal(ind, ind2)
```

Comment:
Note to reviewers: this test has been moved to `sklearn/utils/tests/test_sorting.py` and has been extended.